File: Day26/sum_of_positive_numbers.c
#include<stdio.h>
//Function to find the sum of positive number in a array.
int getSumOfPositiveNumbers(int arr[],int size){
    int sum =0;
    for(int i=0;i<size;i++){
        if(arr[i]>0){
            sum += arr[i];
        }
    }
    return sum; 
}
//Let's Start here
int main(){
    int arr[]={-8,9,4,-3,5,-4,2};
    int size=sizeof(arr)/sizeof(arr[0]);
    int result = getSumOfPositiveNumbers(arr,size);
    printf("\nSum of Positive number: %d\n",result);
    return 0;
}

File: Day26/sumOfPositiveNumbers.js
//Function to find the sum of positive number in a array.
function getSumOfPositiveNumbers(arr){

    let sum =0;
    for(let i=0;i<arr.length;i++){
        if(arr[i]>0){
            sum += arr[i];
        }
    }
    return sum; 
}
//Let's Start here
const arr = [-8,9,4,-3,5,-4,2];
let result = getSumOfPositiveNumbers(arr);
console.log(`Sum of Positive number: ${result}`);
